deltachat-ios/Model/GalleryItem.swift

```diff
@@ -44,8 +44,8 @@ class GalleryItem: ContextMenuItem {
         }
     }
 
-    init(msgId: Int) {
-        self.msg = DcMsg(id: msgId)
+    init(msg: DcMsg) {
```

Comment:
i would have passed context+msgId to the constructor, that way one would stay more flexible and also saves some lines of code on construction.

but this is not a blocker, it is also okay this way, we can change it if it turns out that we need the context.